### PR TITLE
Add Marketplace manifest

### PR DESCRIPTION
**Pull request by jcantrill**
This PR reverts the revert of marketplace manifest

### CODE REVIEW

**Comment by jcantrill** [comment]
/hold

cc @bparees

File: Dockerfile [hunk 1]
```diff
@@ -18,4 +19,5 @@ LABEL io.k8s.display-name="OpenShift elasticsearch-operator" \
       io.k8s.description="This is the component that manages an Elasticsearch cluster on a kubernetes based platform" \
       io.openshift.tags="openshift,logging,elasticsearch" \
       io.openshift.release.operator=true \
```
Comment (bparees) [inline Dockerfile]: you still need to remove this...

Comment (jcantrill) [reply Dockerfile]: https://github.com/openshift/elasticsearch-operator/pull/100

Comment (ewolinetz) [reply Dockerfile]: why is that a separate PR though? it seems like we're going to have an issue if we have both this label and `com.redhat.delivery.appregistry=true` at the same time...

Comment (bparees) [reply Dockerfile]: well the issue will be that this won't pass tess until that line is removed.  :)

**Pull request merged**